src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -535,9 +535,7 @@ public void PublishUnwindInfo(ObjectNode node)
 
                 byte[] blobSymbolName = _sb.Append(_currentNodeZeroTerminatedName).ToUtf8String().UnderlyingArray;
 
-                ObjectNodeSection section = ObjectNodeSection.XDataSection;
-                if (ShouldShareSymbol(node))
-                    section = GetSharedSection(section, _sb.ToString());
+                ObjectNodeSection section = GetSharedSection(ObjectNodeSection.XDataSection, _sb.ToString());
```

Comment:
It looks like LLVM defines a number of different SelectionKinds for COMDAT sections, but I can't see where define any. The LLVM ones look like this: 


[any](https://llvm-swift.github.io/LLVMSwift/Classes/Comdat/SelectionKind.html#/s:4LLVM6ComdatC13SelectionKindO3anyyA2EmF)
[exactMatch](https://llvm-swift.github.io/LLVMSwift/Classes/Comdat/SelectionKind.html#/s:4LLVM6ComdatC13SelectionKindO10exactMatchyA2EmF)
[largest](https://llvm-swift.github.io/LLVMSwift/Classes/Comdat/SelectionKind.html#/s:4LLVM6ComdatC13SelectionKindO7largestyA2EmF)
[noDuplicates](https://llvm-swift.github.io/LLVMSwift/Classes/Comdat/SelectionKind.html#/s:4LLVM6ComdatC13SelectionKindO12noDuplicatesyA2EmF)
[sameSize](https://llvm-swift.github.io/LLVMSwift/Classes/Comdat/SelectionKind.html#/s:4LLVM6ComdatC13SelectionKindO8sameSizeyA2EmF)

Reply:
Reason I'm asking is because it sounds like we would probably only want "any" or "noDuplicates"

Reply:
It appears to do the right thing on Windows, but I haven't checked Linux (hoped I would see it in the CI results).

If it doesn't work on Linux, it's likely because of #77491. I'd back out the non-Windows portion of this change and file a bug to reap this benefit outside Windows in 8.0 (it would be more work than just a one-liner that I randomly tried).